Games/types/Mafia/roles/Mafia/Interceptor.js

```diff
@@ -0,0 +1,12 @@
+const Role = require("../../Role");
+
+module.exports = class Interceptor extends Role {
+
+    constructor(player, data) {
+        super("Interceptor", player, data);
+
+        this.alignment = "Mafia";
+        this.cards = ["VillageCore", "WinWithMafia", "MeetingMafia", "NightTrapper"];
```

Comment:
(if you want to be consistent with EM mechanics)

interceptor is revealed to the rest of the visitors